sdk/communication/arm-communication/README.md

```diff
@@ -63,8 +62,23 @@ const client = new CommunicationServiceManagementClient(new DefaultAzureCredenti
 // });
 // const client = new CommunicationServiceManagementClient(credential, subscriptionId);
 ```
+### Create Communication Service Resource with 'Managed Identity'
 
-
+```javascript
+// Existing resource creation code
+client.communicationServices.beginCreateOrUpdate(resourceGroup, resourceName, resource)
+
+// Create Resource with Managed Identity
+const resource = {
+    location: "Global",
+    dataLocation: "United States",
+    identity: {
+        type: "SystemAssigned" //or UserAssigned
```

Comment:
I'd suggest you create two different resource objects (with different names). That way you  can give the example of system assigned, and one of them with user  assigned. This will make the developers' lives easier 👍